devicetypes/smartthings/ezex-temp-humidity-sensor.src/ezex-temp-humidity-sensor.groovy

```diff
@@ -16,11 +16,12 @@
 import physicalgraph.zigbee.zcl.DataType
 
 metadata {
-    definition(name: "eZEX Temp & Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn:"SmartThings", vid:"generic-humidity") {
+    definition(name: "eZEX Temp & Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn:"SmartThings", vid:"SmartThings-smartthings-SmartSense_Temp/Humidity_Sensor") {
```

Comment:
This UI metadata has battery but this DTH doesn't have that capability

Reply:
@dkirker is that OK?

Reply:
There have no ui metadata without battery, i see this device's UI will display without battery.

Reply:
Might be safer to create a metadata without the battery. I know the plugin tries to be a little smart, but it might be best to not treat that as standard behavior.